ConnectedVessels5_4.py

```diff
@@ -0,0 +1,9 @@
+def interleave(*parameters):
+  listReturn=[]
+  for i,element in enumerate(parameters):
```

Comment:
Please think about changing the names of these variable into more intuitive ones - 
For example, if parameters is actually a list of iterable, perhaps you can change element into iterable.
Also, try not naming your variables with indexes like "i" unless you dont have another option.